index.html

```diff
@@ -1,26 +1,203 @@
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
-  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-
-</head>
-
-<body>
+  <body>
+    <div class="maindiv">
+      <header>
+        <nav class="desktop social-bar">
+          <div class="social-barflex">
+            <i class="fa-brands fa-facebook-f"></i>
+            <i class="fa-brands fa-twitter"></i>
+            <p>English</p>
+            <p>My Page</p>
+            <p>Logout</p>
+          </div>
+        </nav>
+        <nav class="nav">
+          <div class="mobile-nav  mobile">
+          <button class="hamburger-menu">
+            <i class="fa-sharp fa-solid fa-bars fa-3x" id="hamburgericon"></i>
+          </button>
+         </div>
+        </nav>
+        <div id="myNav" class="overlay mobile">
+          <div class="mobile-menu-navbar">
+            <a href="javascript:void(0)" class="closeicon"><i class="closeIcon material-icons  fa-3x"
+                id="close-icon">close</i></a>
+          </div>
+          <div class="overlay-content">
+            <a href="about.html" onclick="closeNav()">About</a>
+            <a href="#project-title" onclick="closeNav()">Program</a>
+            <a href="#about" onclick="closeNav()">Join</a>
+            <a href="#form" onclick="closeNav()">Sponsor</a>
+            <a href="#form" onclick="closeNav()">News</a>
+          </div>
+        </div>
+        <div class="desktop">
+        <div class="desktopmenu">
+          <div>
+            <img 
+              src="./assets/img/gamerzonefooter.png"
+              alt="project1"
+              class="desktopmenuimg"
+            />
+          </div>
+          <div class="menuitemsflex">
+            <a href="about.html">About</a>
+            <a href="#project-title">Program</a>
+            <a href="#about">Join</a>
+            <a href="#form">Sponsor</a>
+            <a href="#form">News</a>
+            <button class="menu-button">BOOK EVENT</button>
+          </div>
+        </div>
+      </div>
+      </header>      
+      <main>
+        <section class="mobile-view leftpad" id="">
+          <div>
+            <p class="firstheader">"Hello Gamers Around The Globe"</p>
+            <h1 class="secondheader">GAMERSZONE ANNUAL EVENT MAINZ 2022</h1>
+          </div>
+          <div class="paragraph-mobile">
+            <p>An annual gathering of all the best gamers every years, meet, challenge
+              learn from the leading Gamers and added tips how to make money by being a gamer. 
+            </p>
+          </div>
+          <div>
+            <h1 class="thirdheader">2022.10.14(THUR) ~ 16(FRI)</h1>
+            <p class="secondparagraph">
+              The next gamerzone event that will be held at Mainz will feature many 
+              activities at the Mainz center  as well as meeting renowned gamers live.
+            </p>
+          </div>
+        </section>
```

Comment:
- [ ] `required`
- [ ] Amazing job so far 👏 
- [ ] however there a few things you need to adjust
- [ ] The `Fonts` and `spacing`
- [ ] please move everything more to the right, 
- [ ] Also try to make the Layout: composition and space between elements, more like the provided design
- [ ] As the requirements states `The pages should look almost identical to the original design.`
- [ ] please see screenshots below for further comparison

`your design`
![image](https://user-images.githubusercontent.com/66230306/210704006-00780cf3-2f82-4939-a67a-83240ea1b878.png)

`Behance design`
![image](https://user-images.githubusercontent.com/66230306/210706604-6607d997-7723-4c67-9532-8de3bb792ad3.png)